Missing testdata/expected_contains.golden file in 1.6.0 relase assets (tar.gz and zip)

### Describe the Bug

The [release assets for 1.6.0](https://github.com/fosrl/newt/releases/tag/1.6.0):
https://github.com/fosrl/newt/archive/refs/tags/1.6.0.zip and https://github.com/fosrl/newt/archive/refs/tags/1.6.0.tar.gz are lacking the internal/telemetry/testdata/expected_contains.golden file. As a result, tests are failing and some systems like nix can't create a package from the source:

```sh
nix-build
[...]
Running phase: checkPhase
--- FAIL: TestMetricsGoldenContains (0.10s)
    telemetry_golden_test.go:60: read golden: open testdata/expected_contains.golden: no such file or directory
FAIL
FAIL    github.com/fosrl/newt/internal/telemetry        0.545s
FAIL
error: Cannot build '/nix/store/g5g698sgwihb0cyiq01n2b6p3z5673fv-newt-1.6.0.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/phbjpn0h6g1hyf4r29cqw7jg127wvpqp-newt-1.6.0
       Last 25 log lines:
       > source root is source
       > Running phase: patchPhase
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > Running phase: buildPhase
       > Building subPackage .
       > Building subPackage ./docker
       > Building subPackage ./healthcheck
       > Building subPackage ./internal/state
       > Building subPackage ./internal/telemetry
       > Building subPackage ./logger
       > Building subPackage ./network
       > Building subPackage ./proxy
       > Building subPackage ./updates
       > Building subPackage ./websocket
       > Building subPackage ./wg
       > package github.com/fosrl/newt/wg: build constraints exclude all Go files in /nix/var/nix/builds/nix-93381-55916061/source/wg
       > Building subPackage ./wgnetstack
       > Building subPackage ./wgtester
       > Running phase: checkPhase
       > --- FAIL: TestMetricsGoldenContains (0.10s)
       >     telemetry_golden_test.go:60: read golden: open testdata/expected_contains.golden: no such file or directory
       > FAIL
       > FAIL  github.com/fosrl/newt/internal/telemetry        0.545s
       > FAIL
       For full logs, run:
         nix log /nix/store/g5g698sgwihb0cyiq01n2b6p3z5673fv-newt-1.6.0.drv
```

### Environment

- OS Type & Version: (e.g., Ubuntu 22.04): N/A
- Pangolin Version: N/A
- Gerbil Version: N/A
- Traefik Version: N/A
- Newt Version: 1.6.0
- Olm Version: (if applicable): N/A


### To Reproduce


```
curl -o https://github.com/fosrl/newt/archive/refs/tags/1.6.0.zip
unzip 1.6.0.zip
cd newt-1.6.0
go test ./...
?   	github.com/fosrl/newt	[no test files]
?   	github.com/fosrl/newt/docker	[no test files]
?   	github.com/fosrl/newt/healthcheck	[no test files]
?   	github.com/fosrl/newt/internal/state	[no test files]
--- FAIL: TestMetricsGoldenContains (0.10s)
    telemetry_golden_test.go:60: read golden: open testdata/expected_contains.golden: no such file or directory
FAIL
FAIL	github.com/fosrl/newt/internal/telemetry	0.613s
?   	github.com/fosrl/newt/logger	[no test files]
?   	github.com/fosrl/newt/network	[no test files]
?   	github.com/fosrl/newt/proxy	[no test files]
?   	github.com/fosrl/newt/updates	[no test files]
?   	github.com/fosrl/newt/websocket	[no test files]
?   	github.com/fosrl/newt/wgnetstack	[no test files]
?   	github.com/fosrl/newt/wgtester	[no test files]
FAIL
```

### Expected Behavior

Go tests succeed

--- a/internal/telemetry/testdata/expected_contains.golden
+++ b/internal/telemetry/testdata/expected_contains.golden
@@ -1,0 +1,3 @@
+newt_connection_attempts_total
+newt_websocket_reconnects_total
+newt_proxy_connections_total

